charts/connect/values.yaml

```diff
@@ -241,6 +241,12 @@ operator:
   # Denotes whether the 1Password Operator will be deployed
   create: false
 
+  # Denotes whether the 1Password Operator will use 1Password Connect to access secrets
+  useWithConnect: true
+
+  # Denotes whether the 1Password Operator will use 1Password Service Account to access secrets
+  useWithServiceAccount: false
```

Comment:
We need to add them in the chart documentation as well.

Reply:
++